### DISCUSSION TITLE
Wizard action outside main div

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#9650] [Help]
In a wizard-type form, is it possible to place the previous, next and save buttons outside the div quadrant that contains the steps and inputs? something like in the following image:

![image](https://github.com/filamentphp/filament/assets/115037170/d6a1b74b-85c8-466d-9fa0-a4b9c476f9cc)

thanks in advanced

**Comment by leandrocfe** [comment]
Customizing the wizard blade file I think
